Makefile

```diff
@@ -76,91 +52,17 @@ coverage:
 	@go tool cover -html=_build/test-coverage-all.out
 
-# get a redis instance up (localhost:1234)
-test-redis:
-	@redis-server --port ${LOCAL_TEST_REDIS_PORT} --daemonize yes; sleep 1
-	@redis-cli -p ${LOCAL_TEST_REDIS_PORT} info > /dev/null
-
-# kill this redis instance (localhost:1234)
-test-redis-kill:
-	@-redis-cli -p ${LOCAL_TEST_REDIS_PORT} shutdown
-
-cross: cross-linux cross-darwin
-
-cross-linux:
-	@mkdir -p ./bin
-	@echo "Building for linux-x86_64..."
-	@env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ./bin/podium-linux-x86_64 ./main.go
-	@$(MAKE) cross-exec
-
-cross-darwin:
-	@mkdir -p ./bin
-	@echo "Building for darwin-x86_64..."
-	@env GOOS=darwin GOARCH=amd64 go build -o ./bin/podium-darwin-x86_64 ./main.go
-	@$(MAKE) cross-exec
-
-cross-exec:
-	@chmod +x bin/*
-
 docker-build:
-	@docker build -t podium .
+	@docker build -f ./build/Dockerfile -t podium .
 
 docker-run:
-	@docker run -i -t --rm -e PODIUM_REDIS_HOST=$(MYIP) -e PODIUM_REDIS_PORT=$(LOCAL_REDIS_PORT) -p 8080:80 podium
+	@docker run -i -t --rm -e PODIUM_REDIS_HOST=$(MYIP) -e PODIUM_REDIS_PORT=6379 -p 8080:80 podium
 
 docker-run-redis:
-	@docker run --name=redis -d -p 6379:$(LOCAL_REDIS_PORT) redis:6.0.9-alpine
+	@docker run --name=redis -d -p 6379:6379 redis:6.0.9-alpine
 
 docker-run-basic-auth:
-	@docker run -i -t --rm -e BASICAUTH_USERNAME=admin -e BASICAUTH_PASSWORD=12345 -e PODIUM_REDIS_HOST=$(MYIP) -e PODIUM_REDIS_PORT=$(LOCAL_REDIS_PORT) -p 8080:80 podium
-
-docker-shell:
-	@docker run -it --rm -e PODIUM_REDIS_HOST=$(MYIP) -e PODIUM_REDIS_PORT=$(LOCAL_REDIS_PORT) --entrypoint "/bin/bash" podium
-
-docker-dev-build:
-	@docker build -t podium-dev -f ./DevDockerfile .
-
-docker-dev-run:
-	@docker run -i -t --rm -p 8080:8080 podium-dev
-
-bench-podium-app: build bench-podium-app-run
-
-bench-podium-app-run: bench-podium-app-kill
-	@rm -rf /tmp/podium-bench.log
-	@./bin/podium start -p 8888 -g 8889 -q -c ./config/perf.yaml 2>&1 > /tmp/podium-bench.log &
-	@echo "Podium started at http://localhost:8888. GRPC at 8889."
-
-bench-podium-app-kill:
-	@-ps aux | egrep 'podium.+perf.yaml' | egrep -v egrep | awk ' { print $$2 } ' | xargs kill -9
-
-# get a redis instance up (localhost:1224)
-bench-redis: bench-redis-kill
-	@redis-server --port 1224 --daemonize yes; sleep 1
-	@redis-cli -p 1224 info > /dev/null
-
-# kill this redis instance (localhost:1224)
-bench-redis-kill:
-	@-redis-cli -p 1224 shutdown
-
-bench-run:
-	@go test -benchmem -bench . -benchtime 5s ./bench/...
-
-bench-seed:
-	@go run bench/seed/main.go
-
-ci-bench-run:
-	@mkdir -p ./bench-data
-	@if [ -f "./bench-data/new.txt" ]; then \
-		mv ./bench-data/new.txt ./bench-data/old.txt; \
-	fi
-	@go test -benchmem -bench . -benchtime 5s ./bench/... > ./bench-data/new.txt
-	@echo "Benchmark Results:"
-	@cat ./bench-data/new.txt
-	@echo
-	@-if [ -f "./bench-data/old.txt" ]; then \
-		echo "Comparison to previous build:" && \
-		benchcmp ./bench-data/old.txt ./bench-data/new.txt; \
-	fi
```

Comment:
Should we delete everything related to benchmark tests? It seems it would be useful to compare our refactor with the previous version. WFYT about extracting these rules into a `BenchmarkMakefile` and importing it to the main `Makefile`?

Reply:
First of all, do these tests work?

Reply:
By talking with @arthur29 we decided that building a load test using Insomnia would be a better approach.